mysql-test/suite/innodb/r/percona_changed_page_bmp.result

```diff
@@ -59,5 +59,53 @@ INSERT INTO t3 VALUES (REPEAT('a', 12582912));
 10th restart
 DROP TABLE t3;
 11th restart
+select * from information_schema.innodb_changed_pages;
+space_id	page_id	start_lsn	end_lsn
+0	0	8204	50037
```

Comment:
I don't think this output is going to be stable, I'd replace this with SELECT COUNT(*) > 0 or similar